app/transports/sse/python_protocol_example.py

```
@@ -0,0 +1,224 @@
+import os
+import json
+from typing import List
+from pydantic import BaseModel
+from dotenv import load_dotenv
+from fastapi import FastAPI, Query
+from fastapi.responses import StreamingResponse
+from openai import OpenAI
+from .utils.prompt import ClientMessage, convert_to_openai_messages
+from .utils.tools import get_current_weather
+
+import json
```

Ruff (0.14.4)

Redefinition of unused `json` from line 2

Remove definition: `json`

(F811)